packages/site/cms-site/src/index.ts

```diff
@@ -9,6 +9,7 @@ export { OneOfBlock } from "./blocks/factories/OneOfBlock";
 export { OptionalBlock } from "./blocks/factories/OptionalBlock";
 export { SeoBlock } from "./blocks/factories/SeoBlock";
 export type { SupportedBlocks } from "./blocks/factories/types";
+export { VideoPreviewImageProps } from "./blocks/helpers/VideoPreviewImage";
```

Comment:
Do we need to export this?

Reply:
I think its good to have necessary interfaces available when creating a custom video preview component